src/Microsoft.TestPlatform.Common/Utilities/RunSettingsProviderExtensions.cs

```diff
@@ -215,7 +215,7 @@ private static XmlNode CreateNode(XmlDocument doc, string xPath)
             return node;
         }
 
-        private static XmlDocument GetRunSettingXmlDocument(this IRunSettingsProvider runSettingsProvider)
+        public static XmlDocument GetRunSettingXmlDocument(this IRunSettingsProvider runSettingsProvider)
```

Comment:
do we need this to be public? Could it be made internal and used via internalsVisibleTo?

Reply:
Class itself is internal. I was using this to get the XML and update it using XmlDocument, not doing that anymore so this is an oversight from me, fixing.